src/nifake/unit_tests/test_session.py

```diff
@@ -832,7 +833,7 @@ def teardown_method(self, method):
     # Session management
 
     def test_init_with_options_and_close(self):
-        session = nifake.Session('dev1', _grpc_channel=object())
+        session = nifake.Session('dev1', _grpc_options=nifake.GrpcSessionOptions(object(), ''))
```

Comment:
You could even pass `None` instead of `object()` for the first session option.

Reply:
Yeah, now that we no longer check if the grpc_channel is truthy, I could change that. I think it's fine this way, too, though.